docs/notebooks/thomson_fitting.ipynb

```diff
@@ -0,0 +1,222 @@
+{
```

Comment:
<p>So if we just had <code>fit_vars = ['n']</code> here, it'd treat <code>Te</code> as a known value, right? (not that we'd want to do that, I'm just not 100% sure how this works from reading the notebook. Could be good to add a 1-element or 3-element (is that even possible?) case for comparison.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PlasmaPy/PlasmaPy/pull/974/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Yes that's correct. </p><p> </p><p>Right now vectors get broken into scalar parameters of the form electron vel = [ev0, ev1, ev2] etc. Handling this in the best way possible is I think the biggest issue I'm struggling with.</p>